app/models/log.rb

```diff
@@ -30,10 +30,19 @@ class Log < ApplicationRecord
 
   def collection_start_year
     return @start_year if @start_year
-    return unless startdate
 
-    window_end_date = Time.zone.local(startdate.year, 4, 1)
-    @start_year = startdate < window_end_date ? startdate.year - 1 : startdate.year
+    if lettings?
+      return unless startdate
+
+      log_start_date = startdate
+    else
+      return unless saledate
+
+      log_start_date = saledate
+    end
```

Comment:
Are you sure that you need to do this? `SalesLog#startdate` is already `saledate`.

Reply:
Ah, I didn't know that, this is probably unnecessary then I agree